docs/advanced-features/react-18/streaming.md

```diff
@@ -68,4 +68,6 @@ Currently, data fetching within `Suspense` boundaries on the server side is not
 
 #### Styling
 
-The Next.js team is working on support for `styled-jsx` and CSS modules in streaming SSR. Stay tuned for updates.
+So far only `styled-jsx` is supported with streaming. The Next.js team is working on support for CSS modules and other CSS solutions in streaming SSR. Stay tuned for updates.
```

Comment:
`unstable_useFlushEffects` should be sufficient for any library that follows the [recommended upgrade guide](https://github.com/reactwg/react-18/discussions/110). It's just that `styled-jsx` support is built-in.